src/view.tree.js

```diff
@@ -23,15 +23,17 @@ class TreeView {
     $jstree.settings.core.data = (node, cb) => {
       const loadAll = this.adapter.canLoadEntireTree() &&
                       this.store.get(STORE.LOADALL)
-      node = !loadAll && (node.id === '#' ? {path: ''} : node.original)
+      // No lazy loading in PR tree view
+      const isPR = this.store.get(STORE.PR) && repo.pullNumber
+      node = !loadAll && !isPR && (node.id === '#' ? {path: ''} : node.original)
```

Comment:
I'm not clear why this is necessary and whether we can simplify or get rid of this?

Reply:
This is necessary because `loadCodeTree` expects `node` to be null if loading the entire tree.  When it constructs the tree view in `_loadCodeTreeInternal` it uses `node` to determine how to display the tree. When `node` is not null, it causes the "_recursive_"  bug that the issue outline.  

We can move that `isPR` check to that function in the adapter. The issue is we still need to check if `isPR` on line 36 so that we can collapse the tree.

Reply:
Ok, thanks. I missed that it was the original logic and you just add the !isPR check to that. Been a while since I looked at this area of the code. I'm gonna merge this fix and see if I can refactor somehow.

Reply:
I refactored a bit https://github.com/buunguyen/octotree/commit/879a2bbd3117f4655118ff66ec1dbae3c8155c7e. Thanks again for working on this!